test/src/e2e_vm_tests/test_programs/should_pass/stdlib/logarythmic_test/Forc.toml

```
@@ -0,0 +1,8 @@
+[project]
+authors = ["Rostyslav Tyshko"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "logarythmic_test"
```

```suggestion
name = "logarithmic_test"
```

You may need to rename your directory etc. if you change this name

---

Fixed